src/oas/oas-to-ms3/oas-30-to-ms3/index.ts

```diff
@@ -32,9 +32,22 @@ class MS3toOAS30toMS3 {
       this.ms3API.securitySchemes = securitySchemasToMS3(this.oasAPI.components.securitySchemes);
     }
 
+    if (this.oasAPI.security) {
+      this.ms3API.settings.securedBy = this.getSecuredBy(this.oasAPI.security);
+    }
+
     return this.ms3API;
   }
 
+  getSecuredBy(securitySchemas: OAS30Interface.SecurityRequirement[]): string[] {
+    const convertedSchemas: string[] = [];
```

Comment:
return   map(securitySchemas, (schema) => {
     const foundSchema = _find(this.ms3API.securitySchemes, {name: Object.keys(schema)[0]});
     return foundSchema.__id;
});